.changeset/late-windows-clean.md

```diff
@@ -0,0 +1,5 @@
+---
+"chainlink": minor
+---
+
+Updated the TXM confirmation logic to use the mined transaction count to identify re-org'd or confirmed transactions. #internal
```

Comment:
This might need to be updated. We'll have to collect all the changes and figure out the impact. For example, we're indirectly deprecating the `ConfirmedMissingReceipt` type.